app/src/components/user/AccountIndexPage/ManageAccount/ActiveLicenseInfo/index.js

```diff
@@ -59,31 +48,8 @@ const ActiveLicenseInfo = ({
     return <React.Fragment></React.Fragment>;
   }
 
-  const renderSessionReplayProSubscription = () => {
-    if (isSessionReplayLifetimeActive) {
-      return (
-        <>
-          <SubscriptionInfo
-            hideShadow={hideShadow}
-            subscriptionDetails={{
-              validFrom: new Date("2023-08-16T00:00:00Z").getTime(),
-              validTill: new Date("2099-08-16T00:00:00Z").getTime(),
-              status: "active",
-              type: "producthunt",
-              planName: "Session Replay Pro",
-            }}
-          />
-          <br />
-        </>
-      );
-    }
-
-    return null;
-  };
-
   return (
     <ProCard title={<h3 style={{ marginBottom: "0" }}>{customHeading}</h3>} className="primary-card github-like-border">
-      {renderSessionReplayProSubscription()}
       {doesSubscriptionExist ? renderSubscriptionInfo() : renderGetASubscription()}
```

Comment:
@RuntimeTerror10 Don't remove this. We still need to show the plan user got during PH launch